photosynthesis_metrics/ssim.py

```diff
@@ -362,7 +369,7 @@ def __init__(self, kernel_size: int = 11, kernel_sigma: float = 1.5, k1: float =
         if scale_weights is None:
             scale_weights_from_ms_ssim_paper = [0.0448, 0.2856, 0.3001, 0.2363, 0.1333]
```

Comment:
All this `self.scale_weights_tensor` and `scale_weights_from_ms_ssim_paper` variables look redundant.
Maybe we can change this to be like:
```python
if scale_weights is None:
    # Scale weight from MS-SSIM paper
    scale_weights = [0.0448, 0.2856, 0.3001, 0.2363, 0.1333]
self.scale_weights = torch.tensor(scale_weights)
#no self.scale_weights_tensor attribute
```
And then use `self.scale_weights` everywhere?

Reply:
I suppose `scale_weights_from_ms_ssim_paper` is just to improve readability.
Regarding `self.scale_weights` and `self. self.scale_weights_tensor`, I'll reduce it to just `self.scale_weights` usage.